data/udisks2.service.in

```diff
@@ -3,10 +3,27 @@ Description=Disk Manager
 Documentation=man:udisks(8)
 
 [Service]
-Type=dbus
 BusName=org.freedesktop.UDisks2
+CapabilityBoundingSet=CAP_SYS_RAWIO
 ExecStart=@udisksdprivdir@/udisksd
+IPAddressDeny=any
 KillSignal=SIGINT
+LimitMEMLOCK=0
+LockPersonality=yes
+MemoryDenyWriteExecute=yes
+NoNewPrivileges=yes
+PrivateNetwork=yes
+PrivateTmp=yes
+ProtectHome=yes
+ProtectSystem=strict
+RestrictAddressFamilies=AF_UNIX AF_NETLINK
+RestrictNamespaces=yes
```

Comment:
Plugins may use namespaces, and for mounting we need the mounts to be available globally. Please remove.

Reply:
This blocks (with seccomp, implying `NoNewPrivileges=yes`) `clone()` system call with `CLONE_NEWIPC` etc. options, which create new namespaces, for example new local mount namespaces. I think these are quite rarely used, only by container managers, systemd itself and hardening software like Firejail. I would not expect plugins to ever do that, it would conflict with the global availability of mounts.